### PR TITLE
Find numbers

### PR DESCRIPTION
**Pull request by SangeetaMishra**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#922

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@SangeetaMishra Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Review by iamrajiv — APPROVED** [review]

File: Python/other/find_the_numbers.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Python program to find the inique numbers in an array out
```
Comment (atarax665) [inline Python/other/find_the_numbers.py]:
```suggestion
# Python program to find the unique numbers in an array out
```

File: Python/other/find_the_numbers.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    x = 0
+    y = 0
+    for i in range(0, n):
+        if arr[i] & xors > 0:
+            x = x ^ arr[i]
+        else:
+            y = y ^ arr[i]
+    if x > y:
+        swap(x, y)
+    print("The distinct numbers are ", x, " and ", y)
```
Comment (atarax665) [inline Python/other/find_the_numbers.py:21]: Add sample Input/Output and Time and Space Complexity in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Review by Abhijit2505 — APPROVED** [review]
Looking good to me, but there are merge conflicts. Please resolve them as well.

**Comment by SangeetaMishra** [comment]
@atarax665 Please review this

**Review by Kajol-Kumari — APPROVED** [review]
LGTM :+1:

**Pull request merged**